data/dynamodb.go

```diff
@@ -196,6 +202,19 @@ func createTableIfNotExists(
 		return err
 	}
 
+	_, err = client.UpdateTimeToLive(&dynamodb.UpdateTimeToLiveInput{
+		TableName: aws.String(cfg.Table),
+		TimeToLiveSpecification: &dynamodb.TimeToLiveSpecification{
+			AttributeName: aws.String(cfg.TTLAttributeName),
+			Enabled:       aws.Bool(true),
+		},
+	})
+
+	if err != nil && !strings.Contains(err.Error(), "already enabled") {
```

Comment:
shouldn't this be `strings.Contains(err.Error(), "already enabled")`?

Reply:
The only allowed error is if TTL attr already exists